src/Microsoft.Win32.Registry/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <PackageConfigurations>
       netstandard;
+      netstandard-Windows_NT;
```

Comment:
Why do we need this netstandard-Windows_NT configuration?

Reply:
Because since we harvest a Windows specific netstandard1.3, then package validation will complain given that the compile asset resolved for UAP6.0 (the netstandard2.0 ref) will have a higher assembly version than the resolved implementation for UAP6.0 (the windows specific netstandard1.3 implementation) because the runtime specific assets will be prefered over the non rid specific (lib/netstandard2.0) . This is required in order to pass package validation and in order to be consumed on UAP.

Reply:
@ericstj do you think adding the netstandard-Windows_NT configuration is better then remapping the harvested location to put the NS13 asset in the lib instead of the windows RID. I suppose there is a chance that remapping the path might actually cause breaks but it would depend on the actual package.

Reply:
That ns1.3 asset is a *windows specific implementation*.  If you started putting that in a RID-less folder you'd change the behavior for non-windows (previously it got a PNS exception).

If you go back and look at the history as to why this is targeting NETCoreapp-window_nt today instead of netstandard-windows_nt, it is because of this: https://github.com/joperezr/corefx/commit/28097ca87cc2e8bcf7760f05732464b2c00eeaa4#diff-6b3a566cbc8f4a17c79fa82c0a0feafc.  I needed to create a netcoreapp configuration to avoid an assembly cycle in the implementation of netcoreapp.

In that case the better fix here would be to move the netcoreapp-* configurations into BuildConfigurations only (since their only difference from netstandard version will be referenced assemblies) and have a netstandard-windows_NT version in the package.